dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -882,7 +917,7 @@ private fun makeCommonSqlToKTypeMapping(tableColumnMetadata: TableColumnMetadata
         Types.DOUBLE to Double::class,
         Types.NUMERIC to BigDecimal::class,
         Types.DECIMAL to BigDecimal::class,
-        Types.CHAR to Char::class,
+        Types.CHAR to String::class,
```

Comment:
So we cannot parse `char(1)` as Char? just `char(n)` as String

Reply:
May be for the best probably actually, if we cannot check `n`

Reply:
Because JDBC has no special readChar/writeChar methods and returns String value for Char(1), we leave for a user the decision to convert.